code/game/gamemodes/umbrage/umbrage_abilities/silver_tongue.dm

```diff
@@ -9,6 +9,17 @@
 	lucidity_cost = 1 //Very niche, so low cost
 	blacklisted = 0
 
+/datum/action/innate/umbrage/New()
+	START_PROCESSING(SSobj, src)
+	..()
+
+/datum/action/innate/umbrage/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	return ..()
+
+/datum/action/innate/umbrage/process()
+	UpdateButtonIcon() //Constantly checks the shuttle call
```

Comment:
it should only update if the shuttle mode has changed to/from SHUTTLE_CALL (track this with a var)